pkgs/development/python-modules/radian/default.nix

```
@@ -19,7 +19,7 @@
 
 buildPythonPackage rec {
   pname = "radian";
-  version = "0.6.12";
+  version = "0.6.13";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `build-system = [ setuptools ];` to avoid the legacy build.

see https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters